button/button.css

```diff
@@ -0,0 +1,191 @@
+.button-intro {
+  font-size: 1.25rem;
+  padding: 1rem 0 1rem 0;
+}
+.button-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2rem;
+  height: 16rem;
+  margin-bottom: 1rem;
+}
+
+/* solid buttons */
+
+.ranger-primary-button-sm,
+.ranger-primary-button-md,
+.ranger-primary-button-lg,
+.ranger-btn-with-icon-button-sm,
+.ranger-btn-with-icon-button-md,
+.ranger-btn-with-icon-button-lg {
+  border: none;
+  color: var(--white1);
+  border: none;
+  font-size: 1.25rem;
+}
+
+.ranger-primary-button-sm {
+  background-color: var(--blue2);
+  width: 8rem;
+  padding: 1rem 2rem;
+}
+.ranger-btn-with-icon-button-sm {
+  background-color: var(--blue2);
+  width: 10rem;
+  padding: 1rem;
+}
+.ranger-primary-button-md,
+.ranger-btn-with-icon-button-md {
+  background-color: var(--yellow2);
+  width: 12rem;
+  padding: 1rem 2rem;
+}
+
+.ranger-primary-button-lg,
+.ranger-btn-with-icon-button-lg {
+  background-color: var(--orange2);
+  width: 16rem;
+  padding: 1.5rem 2rem;
+}
+
+.ranger-primary-button-sm:hover,
+.ranger-primary-button-md:hover,
+.ranger-primary-button-lg:hover,
+.ranger-btn-with-icon-button-sm:hover,
+.ranger-btn-with-icon-button-md:hover,
+.ranger-btn-with-icon-button-lg:hover {
+  box-shadow: 0 1px 2px 0 rgb(0 0 0 / 25%);
+}
+
+/* Outlined buttons */
+.range-secondary-button-sm,
+.range-secondary-button-md,
+.range-secondary-button-lg {
+  border: 2px solid;
+  background-color: transparent;
+  font-size: 1.25rem;
+}
+.range-secondary-button-sm {
+  border-color: var(--greyFour);
```

Comment:
Maintain a single way of naming colors. Prefer to use shade names rather than numbers.